mindmeld/components/_config.py

```diff
@@ -455,6 +456,11 @@
     },
 }
 
+DEFAULT_TOKENIZER_CONFIG = {
+    # populated in the `get_tokenizer_config` func
+    "allowed_patterns": [],
```

Comment:
can we call directly here:
```
"allowed_patterns": _get_default_regex(exclude_from_norm)

Reply:
I just did that to preserve order, otherwise the function will need to move above this config. That works too.

Reply:
Just realised that it uses a parameter that's coming from the tokenizer class, so can't change this.